code/obj/item.dm

```diff
@@ -496,6 +507,13 @@
 						C.expose()
 				the_dir = turn(the_dir,45) */
 
+/obj/item/proc/combust_ended()
+	processing_items.Remove(src)
```

Comment:
Uh oh, this is not a good idea. What if the item is in `processing_items` just by itself, not just because of fire.

Reply:
i added a workaround to stop processing items from combusting altogether. it's a little hacky, but there really isnt a framework for associating items in the processing list with what they're being processed for...

Honestly i just didn't feel like messing with another core system